src/test/java/io/vertx/test/core/HttpDecompressionTest.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2011-2014 The original author or authors
+ * ------------------------------------------------------
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Apache License v2.0 which accompanies this distribution.
+ *
+ *     The Eclipse Public License is available at
+ *     http://www.eclipse.org/legal/epl-v10.html
+ *
+ *     The Apache License v2.0 is available at
+ *     http://www.opensource.org/licenses/apache2.0.php
+ *
+ * You may elect to redistribute this code under either of these licenses.
+ */
+
+package io.vertx.test.core;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpClientOptions;
+import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.HttpServerOptions;
+
+import java.io.ByteArrayOutputStream;
+import java.util.zip.GZIPOutputStream;
+
+import org.junit.Test;
+
+/**
+ * @author <a href="mailto:clonyara@gmail.com">Yan Gleyzer</a>
+ */
+public class HttpDecompressionTest extends HttpTestBase {
+
+  public void setUp() throws Exception {
+    super.setUp();
+    client = vertx.createHttpClient(new HttpClientOptions());
+    server = vertx.createHttpServer(new HttpServerOptions().setPort(DEFAULT_HTTP_PORT).setDecompressionSupported(true));
+  }
+
+  @Test
+  public void testDefaultRequestHeaders() throws Exception {
```

Comment:
I put it in separate file just like `io.vertx.test.core.HttpCompressionTest`.
I can move to `Http1xTest`, of course